### ISSUE TITLE
WP Rocket conflict wit Jet Reviews 2.0 Beta

### ISSUE THREAD

**Issue by colderik** [Crocoblock/suggestions#1641]
I have JetReviews 2.0 Beta configured on a CPT (within a Elementor Pro template). When I activate WP Rocket the front end shows a spinning wheel where review widget should be. When I deactivate WP Rocket it displays correctly.

More specifically, the issue arises when one or both of the following options are enabled:

- Minify JavaScript files
- Load JavaScript deferred

Screenshot shows options deselected.

I can provide errors from Chrome console if required.
<img width="615" alt="Screenshot 2020-07-25 at 01 33 42" src="https://user-images.githubusercontent.com/28677098/88444941-8fc76300-ce17-11ea-9267-a5972a0babeb.png">

**Comment by Grassmedia** [comment]
Exactly the same issue with Cloudways Breeze caching plugin.

**Comment by EquilibriumTechx** [comment]
Yes we, are also having the same issue, the only workaround I can find is to ignore this URL which has jet reviews. 
by adding it to set "never cache this URL" in  the advance rules section of 
![Screenshot 2021-01-06 at 12 14 46 AM](https://user-images.githubusercontent.com/65528418/103670307-5f821400-4fb4-11eb-95f9-7109796919cf.png)

We have tried to exclude the JS assets and also CSS, the issue is presistent , if anyone has a better solution please share them

**Comment by pamanseau** [comment]
We have the same issue with LiteSpeed Cache

Seems to be a problem with the uniqueid

Defered or Delay is not working but as well as minify/combine. One or the other is giving the issue.

I tried to exclude inline script at line 157 of jet-reviews/includes/components/reviews/render/review-listing-render.php
`<script id="<?php echo 'jetReviewsWidgetOptions' . $uniqid ?>" type="text/javascript" data-no-optimize="1" data-no-defer="1">`

Tried to exclude js scripts from minify:
jet-reviews-frontend.js

It gave an error for vue() not define

excluded vue.js and vue.min.js from Minify but got the same issue

Caching works with everything else but Jet Reviews. it's unfortunate for websites like ours that drive on user reviews.
Hope there will be a solution for this.

**Comment by EquilibriumTechx** [comment]
Sadly crocolock is not paying attention to this matters. I have repeatedly mention this to them over chat and support emails. and they dont seem to care. and it's been 1 year ago since I reported. still very sad, there is no fixed for it, what I did is to get another review plugin and that solves the problem, and just avoided this plugin. since its broken

**Comment by vincentchu-fr** [comment]
same issue with litespeed cache. jet reviews doesn't show up.

**Comment by imihaiart** [comment]
The issue is still here! Jet Reviews cannot be used with Litespeed Cache!
**Very sad after working more hours on building and customizing the reviews section!**

Can't believe the Crocoblock devs never though we'll ever use Jet Reviews along with caching plugins.

**Comment by EquilibriumTechx** [comment]
Ahh I see someone is being assigned to this case, finally some responds
